base-helm-configs/kube-ovn/kube-ovn-helm-overrides.yaml

```diff
@@ -25,7 +25,8 @@ MASTER_NODES: ""
 MASTER_NODES_LABEL: "kube-ovn/role=master"
 
 networking:
-  # NET_STACK could be dual_stack, ipv4, ipv6
+  # NET_STACK defaults to ipv4, other examples like
+  # dual_stack, ipv6 are commented out
```

Comment:
"commented out" doesn't mean ignored from the helm point of view, it just means its using the defaults.

Reply:
That's fine I like to remove that, reduces clutter